tests/integration/asserts/time.py

```diff
@@ -52,6 +52,18 @@ def _truncate(dt, resolution=None):
     }[resolution](dt)
 
 
+def _format_resolution(dt: datetime, resolution):
+    match resolution:
+        case Resolution.Seconds:
+            return str(dt.timestamp())
+        case Resolution.MilliSeconds:
+            return str(dt.timestamp() * 1_000)
+        case Resolution.MicroSeconds:
+            return str(int(dt.timestamp() * 1_000_000))
+        case Resolution.NanoSeconds:
+            return str(int(dt.timestamp() * 1_000_000_000))
```

Comment:
Not a big deal, but you use scientific notation for this in `test_ourlogs.py`.

Reply:
Now that's gone too.